.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Perhaps we can add this file to `.gitignore`

Reply:
It was already in .gitignore. Possible added accidentally into the repository before was in gitignore. I have removed the file.